src/Cafe/HW/Latte/Core/LatteShaderCache.cpp

```diff
@@ -484,6 +541,36 @@ void LatteShaderCache_ShowProgress(const std::function <bool(void)>& loadUpdateF
 			lastFrameUpdate = tick_cached();
 		}
 
+				g_renderer->BeginFrame(false);
+				if (g_renderer->ImguiBegin(false)) {
+				ImGui::SetNextWindowPos({ 0,0 }, ImGuiCond_Always);
+				ImGui::SetNextWindowSize(io.DisplaySize, ImGuiCond_Always);
+				ImGui::PushStyleVar(ImGuiStyleVar_WindowBorderSize, 0);
+				ImGui::PushStyleVar(ImGuiStyleVar_WindowPadding, { 0,0 });
+				if (ImGui::Begin("Background texture2", nullptr, kPopupFlags))
+				{
+					if (d_shaderCacheLoaderState.textureId)
+				{
```

Comment:
The `if` and it's `{` should be at the same indentation. But also the entire block is too far indented.
I noticed that in general there is a bit of weird formatting. Try to match the coding style of the surrounding code :)

Reply:
Probably Just VS code acting weird.

Reply:
Anyways I tried fixing it as best as I can